### ISSUE TITLE
Links with underscores

### ISSUE THREAD

**Issue by hipstersmoothie** [coding-intl/micro-down#7]
The following markdown produces italics in the link text.

```md
[https://en.wikipedia.org/wiki/Open_Financial_Exchange](https://en.wikipedia.org/wiki/Open_Financial_Exchange)
```

<img width="382" alt="screen shot 2018-10-18 at 12 17 37 am" src="https://user-images.githubusercontent.com/1192452/47137380-3fe36180-d26b-11e8-8bd2-c93e79a8e88a.png">

Github's rendering:

[https://en.wikipedia.org/wiki/Open_Financial_Exchange](https://en.wikipedia.org/wiki/Open_Financial_Exchange)

**Comment by ShynRou** [comment]
@hipstersmoothie Thanks for the report, your are right there should not be inline styling in link texts.
I'll take a look at it.

**Comment by ShynRou** [comment]
Just published 1.5.0 
I reworked the link, image, code and iframe handling.
Now you can use it as in GitHub only with external styling
 _[https://en.wikipedia.org/wiki/Open_Financial_Exchange](https://en.wikipedia.org/wiki/Open_Financial_Exchange)_
